package.json

```diff
@@ -343,8 +344,8 @@
   "dependencies": {
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
     "@glif/filecoin-wallet-provider": "2.0.0-alpha.14",
-    "@ledgerhq/hw-app-eth": "6.24.1",
-    "@ledgerhq/hw-transport-webhid": "6.24.1",
+    "@ledgerhq/hw-app-eth": "6.22.3",
+    "@ledgerhq/hw-transport-webhid": "6.20.0",
```

Comment:
why were these versions lowered?

Reply:
I will revert this to match those on `master`.